internal/Picker/tests/Picker-specs.js

```diff
@@ -1,35 +1,34 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {fireEvent, render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
 import Picker from '../Picker';
 import PickerItem from '../PickerItem';
-import css from '../Picker.module.less';
 
-const tap = (node) => {
-	node.simulate('mousedown');
-	node.simulate('mouseup');
-};
-const decrement = (slider) => tap(slider.find('IconButton').last());
-const increment = (slider) => tap(slider.find('IconButton').first());
+const increment = (slider) => userEvent.click(slider.firstElementChild);
+const decrement = (slider) => userEvent.click(slider.lastElementChild);
 
 describe('Picker Specs', () => {
-
 	test('should have a default \'value\' of 0', () => {
-		const picker = mount(
+		render(
```

Comment:
Please write all the render methods on a single line(as long as it has no children or it fits).